db/seeds.d/189_add_host_inventory_param.rb

```diff
@@ -0,0 +1,7 @@
+CommonParameter.without_auditing do
```

Comment:
Is there any problem with this param living in foreman_rh_cloud but `host_registration_insights` living in foreman? I don't fully get the history or connection.

Reply:
That's OK. The `host_registration_isnights` is used by the registration process to install `insights-client` and we were using it to indicate the need to upload to the cloud. Now that we split the roles, one can live in core and the other here.